src/components/navbar/navbar.tsx

```diff
@@ -0,0 +1,90 @@
+import { type FC, useState } from "react";
+import { Button, Stack, Typography } from "@mui/material";
+import { signOut } from "next-auth/react";
+import { useRouter } from "next/router";
+
+import { text } from "~/assets/text/index.js";
+import { styles } from "./styles.js";
+import {
+  BotBustersIcon,
+  SoundOffIcon,
+  SoundOnIcon,
+  UserIcon,
+} from "~/assets/icons";
```

Comment:
```suggestion
} from "~/assets/icons/index.js";
```
at least I believe so lol